<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>538</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>798</width>
    <height>538</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>798</width>
    <height>538</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Event Driven Programming Project</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>House View</string>
     </attribute>
     <attribute name="toolTip">
      <string>View of entire house</string>
     </attribute>
     <widget class="QLabel" name="mapImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>491</width>
        <height>441</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Room View</string>
     </attribute>
     <widget class="QLabel" name="roomNameLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>451</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Inventory</string>
     </attribute>
     <widget class="QLabel" name="inventoryDisplay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>501</width>
        <height>451</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="item1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="item2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="item3">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="item4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="item5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>160</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="item6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>160</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="item7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="item8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>310</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="item9">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>310</y>
        <width>111</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>251</width>
      <height>135</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="goButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Moves character to the selected room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Go to Room</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Moves character to the selected room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Go Upstairs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Moves character to the selected room&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Go Downstairs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SearchButton">
       <property name="text">
        <string>Search Room</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="exitsLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Exits:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="roomInfoOutput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
